osidb/models.py

```diff
@@ -2219,11 +2219,12 @@ class Meta:
 
         # Ensure that it's not possible to have two bzimports running concurrently
         # and succeed while creating numbering conditions impossible to handle later.
-        constraints = [
-            models.UniqueConstraint(
-                fields=["flaw", "order"], name="unique_per_flaw_comment_nums"
-            ),
-        ]
+        # TODO: Re-enable this constraint once the "order" field is removed from the model.
```

Comment:
We can't re-enable the constraint if we introduce comments that lack the order, or have duplicate ordering, or it will be difficult to do so.

This effectively changes the API because the client can't use the `order` field for sorting comments anymore, and I don't see any sorting being used when serializing on OSIDB side other than the model's `Meta` which will necessarily cause very out-of-chronological-ordering comment order, which is bound to be very confusing:

```
    class Meta:
        """define meta"""

        ordering = (
            "order",
            "external_system_id",
            "uuid",
            "created_dt",
        )
```

The commit with message `Make order field optional` doesn't explain why.

Reply:
What I wrote ^ is incorrect.

We had a call. Thank you for explaining the changes and changes in Django. I was confused and not knowledgeable enough (and didn't have the time to do my own research without blocking it even longer). For posterity, the `order` field is still being set by `FlawCommentSerializer.create()`, so it will continue behaving exactly as it used to behave.